### ISSUE TITLE
C-w (jump to source) does not work.

### ISSUE THREAD

**Issue by AndreaOrru** [Sarcasm/company-irony#24]
See title.

As an example, company-semantic provides this feature.
Is it because of some clang limitation?

**Comment by Sarcasm** [comment]
Yeah, when I initially implemented the feature the information was not easily accessible.
Things may have changed but I doubt it.
